### ISSUE TITLE
ptt hardware buttons not working when screen is off

### ISSUE THREAD

**Issue by kubbur** [acomminos/Plumble-Legacy#58]
Works when screen is on, not when screen is off
Phone model n7100, stock (unrooted)

**Comment by Zuko** [comment]
Its normal
 21 sty 2013 20:56, "kubbur" notifications@github.com napisał(a):

**Comment by kubbur** [comment]
Possible to implement in near future?
Would mean the world to me.

**Comment by acomminos** [comment]
Sadly not possible. We can only intercept key events when the activity is in the foreground. Would love to have this, though.

**Comment by kubbur** [comment]
Not true, this is possible through device admin and/or wakelock or content observer
stackoverflow.com/questions/6896746/android-is-there-a-broadcast-action-for-volume-changes
Even if it would drain the battery it would be a huge success, could just add a tic box for it

**Comment by acomminos** [comment]
That's only for one variety of push to talk setup, with volume buttons- I meant in general. Still, I could potentially add this as an option if the user selects a volume button when prompted.

**Comment by acomminos** [comment]
You're right though, a wakelock allowing the screen to turn off but not to lock would work fine. I'll definitely throw that in.

**Comment by kubbur** [comment]
Most phones come with a headset with remote controls, the volume buttons on it bear the same tags as on the phone thus making it super comfy to use them as remotes for just about anything, and combined with wakelock i could use the ptt from the pocket, also, wakelock would be useful for not letting the phone cut on the 3g/wireless
